packages/components/src/components/composite-checkbox/composite-checkbox.jsx

```diff
@@ -44,7 +44,7 @@ CompositeCheckbox.propTypes = {
     label: PropTypes.string.isRequired,
     id: PropTypes.string,
     description: PropTypes.string.isRequired,
-    children: PropTypes.oneOfType(PropTypes.node, PropTypes.arrayOf(PropTypes.node)),
+    children: PropTypes.oneOfType([PropTypes.node, PropTypes.arrayOf(PropTypes.node)]),
```

Comment:
fix for the console.error '_Warning: Invalid argument supplied to oneOfType, expected an instance of array.'_